File: MCP_Backend-master/src/main/java/com/greybox/mediums/models/InterswitchPaymentItem.java
package com.greybox.mediums.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class InterswitchPaymentItem {
    private Integer categoryid;
    private Integer billerid;
    private boolean isAmountFixed;
    private Integer paymentitemid;
    private String paymentitemname;
    private BigDecimal amount;
    private String code;
    private String currencyCode;
    private String currencySymbol;
    private String itemCurrencySymbol;
    private String sortOrder;
    private String pictureId;
    private String paymentCode;
}

File: micropay-mobile-apis-master/Models/InterswitchPaymentItem.cs
using System.Collections.Generic;

namespace micropay_apis.Models
{
    public class InterswitchPaymentItem
    {
        public string categoryid { get; set; }
        public int? billerid { get; set; }
        public bool isAmountFixed { get; set; }
        public int? paymentitemid { get; set; }
        public string paymentitemname { get; set; }
        public double? amount { get; set; }
        public string code { get; set; }
        public string currencyCode { get; set; }
        public string currencySymbol { get; set; }
        public string itemCurrencySymbol { get; set; }
        public string sortOrder { get; set; }
        public string pictureId { get; set; }
        public string paymentCode { get; set; }
    }

    public class PaymentResponse
    {
      public List<InterswitchPaymentItem> paymentitems { get; set; }
    }
}